src/encoded/static/components/summary.js

```diff
@@ -345,45 +326,51 @@ SummaryData.defaultProps = {
 class SummaryBody extends React.Component {
     constructor(props) {
         super(props);
+        this.state = {
+            CellCount: 0
+        }
         const searchQuery = url.parse(this.props.context['@id']).search;
         const terms = queryString.parse(searchQuery);
-        this.state = {
-            selectedOrganism: terms[organismField] ? terms[organismField] : [],
-        };
-        this.chooseOrganism = this.chooseOrganism.bind(this);
     }
 
-    chooseOrganism(e) {
-        this.setState({
-            selectedOrganism: e.currentTarget.id,
-        });
-        const parsedUrl = url.parse(this.props.context['@id']);
-        const query = new QueryString(parsedUrl.query);
-        query.deleteKeyValue(systemsField);
-        query.deleteKeyValue(organField);
-        query.replaceKeyValue(organismField, e.currentTarget.id, '');
-        const href = `?${query.format()}`;
-        this.context.navigate(href);
+    componentDidMount() {
+        const query_url = this.props.context.search_base.replace('/search/?', '')
+        this.getCellCount(query_url);
+    }
+
+    getCellCount(searchBase) {
```

Comment:
no problem with the code, but I think we should move this to the backend in the future and have this calculation done and returned in the API